lambda/models/state_machine/create_model.py

```diff
@@ -14,27 +14,77 @@
 
 """Lambda handlers for CreateModel state machine."""
 
+import json
+import os
 from copy import deepcopy
 from typing import Any, Dict
 
+import boto3
+from botocore.exceptions import ClientError
+from models.domain_objects import CreateModelRequest
+
+lambdaClient = boto3.client("lambda")
+ecrClient = boto3.client("ecr")
+ec2Client = boto3.client("ec2")
+ddbResource = boto3.resource("dynamodb")
+model_table = ddbResource.Table(os.environ["MODEL_TABLE_NAME"])
+
 
 def handle_set_model_to_creating(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Set DDB entry to CREATING status."""
     output_dict = deepcopy(event)
-    output_dict["create_infra"] = True
+    request = CreateModelRequest.validate(event)
+
+    if (
+        request.AutoScalingConfig
```

Comment:
depending on the formatting, we can try to put the `and` at the end of each line with a continuation slash if needed, or we can put a comment to have the formatter ignore this situation specifically `#noqa: W503` or something like that

Reply:
no matter how I format it, black will change the formatting to what I currently have, then flake8 will come along and fail afterwards

Reply:
changed to use inline ignores instead of global